docs/excel/excel-add-ins-comments.md

```diff
@@ -197,8 +197,121 @@ Excel.run(function (context) {
 });
 ```
 
+## Comment events
+
+Your add-in can listen for comment additions, changes, and deletions. [Comment events](/javascript/api/excel/excel.commentcollection#event-details) occur on the `CommentCollection` object. To listen for comment events, register the `onAdded`, `onChanged`, or `onDeleted` comment event handler. When a comment event is detected, use this event handler to retrieve data about the added, changed, or deleted comment. The `onChanged` event also handles comment reply additions, changes, and deletions. 
+
+See the [Work with Events using the Excel JavaScript API](excel-add-ins-events.md) article for additional information about registering event handlers, handling events, and removing event handlers. 
+
+### Comment addition events 
+The `onAdded` event is triggered when a new comment is added to the comment collection. This event isn't triggered when replies are added to a comment thread. See [Comment change events](#comment-change-events) to learn about comment reply events.
```

Comment:
A couple suggestions:
- Make it clear this event works on multiple comments at a time.
- Consider making the reply part as more of a note about the first sentence.
```suggestion
The `onAdded` event is triggered when one or more new comments are added to the comment collection. This event is *not* triggered when replies are added to a comment thread (see [Comment change events](#comment-change-events) to learn about comment reply events).
```